src/components/Person/Person.jsx

```diff
@@ -1 +1,25 @@
-// Write code here
+/* eslint-disable react/jsx-filename-extension */
+import React from 'react';
+import './Person.scss';
+
+export function Person({
+  name,
+  age,
+  sex,
+  isMarried,
+  partnerName,
+}) {
+  const partner = (sex === 'm')
+    ? (`My wife's name is ${partnerName}`)
+    : (`My husband's name is ${partnerName}`);
+
+  return (
+    <div className="Person">
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
+      {age ? (<p className="Person__age">{`I am ${age}`}</p>) : null}
```

Comment:
Let's not write code in 1 line, format it properly. And you can also use `&&` operator here, cause you either render or don't render something